swiftui-project/ProductDetailsView.swift

```diff
@@ -0,0 +1,118 @@
+                                .foregroundColor(.black)
+                            
+                        }
+                        .padding(.horizontal, 10)
+                        .padding(.vertical, 10)
+                        .background(Color.white)
+                        .cornerRadius(26)
+                        .padding(.vertical,15)
+                        .padding(.horizontal, 30)
+                    })
+                }
+                
+                
+                VStack(alignment: .leading) {
+                    HStack(spacing:1){
+                        Text("Stargazer's Tea Set")
+                            .font(.title3)
+                            .bold()
+                        Spacer()
+                        Text("5")
+                            .font(.headline)
+                            .bold()
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
+                            .foregroundColor(Color(UIColor(named: "TextColor")!))
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
+                            .foregroundColor(Color(UIColor(named: "TextColor")!))
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
+                            .foregroundColor(Color(UIColor(named: "TextColor")!))
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
+                            .foregroundColor(Color(UIColor(named: "TextColor")!))
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
+                            .foregroundColor(Color(UIColor(named: "TextColor")!))
+                    }
+                    
+                    Text("Category: Home")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                    
+                    Text("Indulge in a heavenly tea experience with our Stargazer’s Tea Set, featuring a constellation-themed teapot and matching teacups. Crafted from fine porcelain, this elegant set will transport you to the cosmos with every sip.")
+                        .font(.body)
+                        .lineLimit(nil)
+                        .padding(.vertical)
+                        .frame(width: 300)
+                    
+                    Text("$90.00")
+                        .font(.title)
+                        .bold()
+                        .padding(.vertical)
+                }
+                .padding(.horizontal,20)
+                .padding(.bottom,50)
+                
+                Button(action: {}, label: {
+                    Text("Add to cart")
+                        .foregroundColor(.white)
+                        .frame(width: 340, height: 46)
+                        .background(Color(UIColor(named: "TextColor")!))
```

Comment:
You don't need need to use UIColor - Color("TextColor") should be work

Reply:
Fixed.